src/actions/sources/prettyPrint.js

```diff
@@ -124,6 +126,8 @@ export function togglePrettyPrint(sourceId: string) {
       })
     );
 
+    dispatch(closeTab(source));
+    dispatch({ type: "TOGGLE_PRETTY_PRINTING" });
```

Comment:
hmm, could we hook into the fact that the pretty printed source is loading and use that field?

https://github.com/devtools-html/debugger.html/blob/master/src/actions/sources/prettyPrint.js#L35-L44

Reply:
So, the Footer component doesn't know about that, and I think it would be hacky if we try to find what is the new pretty source and it it's loading or loaded?